client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCart.tsx

```diff
@@ -32,12 +33,17 @@ const AdminCart = ({
   const itemsInCart = cartItems.map((item, index) => (
     <p
       key={index}
-      className='flex flex-row gap-8 bg-gradient-to-b from-zinc-700 px-5 py-3 rounded-xl mb-5'
+      className='flex flex-row flex-wrap bg-gradient-to-b from-zinc-700 px-5 pt-3 pb-4 rounded-xl mb-5'
     >
-      <span className='flex-auto text-left'>
+      <span className='flex-1 text-left font-bold'>
         {item.qty} x {item.name}
       </span>
-      <span className='flex-auto text-right'>{toDollar(item.price)}</span>
+      <span className='flex-1 text-right font-bold'>
+        {toDollar(item.price)}
+      </span>
+      <span className='w-full text-left text-xs'>
+        {item.desc}
+      </span>
     </p>
```

Comment:
I imagine that the `YourOrder` version of this component should exactly mirror the html structure of this component